src/ui/flutter_app/lib/l10n/intl_es.arb

```diff
@@ -404,23 +404,23 @@
     "@alternate": {
         "description": "Alternate"
     },
-    "isAutoRestart": "Reinicio automático del juego al terminar la partida",
+    "isAutoRestart": "Autoreiniciar del juego al terminar la partida",
     "@isAutoRestart": {
         "description": "Auto-restart game when game over"
     },
-    "isAutoChangeFirstMove": "Cambiar automáticamente el primer movimiento",
+    "isAutoChangeFirstMove": "Autocambiar el primer movimiento",
     "@isAutoChangeFirstMove": {
         "description": "Auto Change First Move"
     },
     "resignIfMostLose": "Renuncia de la IA si la mayoría pierde",
     "@resignIfMostLose": {
         "description": "AI Resign if Most Lose"
     },
-    "shufflingEnabled": "Movimiento aleatorio",
+    "shufflingEnabled": "Mover al azar",
     "@shufflingEnabled": {
         "description": "Random move"
```

Comment:
This translation is more accurate, and I will amend the English translation accordingly later to:  `Move randomly`.

Reply:
Yes. Randomly is "al azar".